include/config/config_graphics.h

```diff
@@ -136,12 +136,13 @@
 
 
 /**
- * Limits the horizontal fov on emulator like on console. May break viewport widescreen hacks.
+ * May break viewport widescreen hacks.
+ * When this is disabled,the culling will only be skipped according to the NO_CULLING_EMULATOR_BLACKLIST
```

Comment:
I'm usually not this anal when it comes to conventions, but for actual config files, there should be no typos ideally. (Please add a space after the comma and add a period at the end of the second line to be consistent with the line above.)